api/handlers/container/create.go

```diff
@@ -172,15 +250,19 @@ func (h *handler) create(w http.ResponseWriter, r *http.Request) {
 		GOptions: globalOpt,
 
 		// #region for basic flags
-		Interactive: false,                     // TODO: update this after attach supports STDIN
-		TTY:         false,                     // TODO: update this after attach supports STDIN
-		Detach:      true,                      // TODO: current implementation of create does not support AttachStdin, AttachStdout, and AttachStderr flags
-		Restart:     restart,                   // Restart policy to apply when a container exits.
-		Rm:          req.HostConfig.AutoRemove, // Automatically remove container upon exit.
-		Pull:        "missing",                 // nerdctl default.
-		StopSignal:  stopSignal,
-		StopTimeout: stopTimeout,
-		CidFile:     req.HostConfig.ContainerIDFile, // CidFile write the container ID to the file
+		Interactive:        false,                     // TODO: update this after attach supports STDIN
+		TTY:                false,                     // TODO: update this after attach supports STDIN
+		Detach:             true,                      // TODO: current implementation of create does not support AttachStdin, AttachStdout, and AttachStderr flags
+		Restart:            restart,                   // Restart policy to apply when a container exits.
+		Rm:                 req.HostConfig.AutoRemove, // Automatically remove container upon exit.
+		Pull:               "missing",                 // nerdctl default.
```

Comment:
Unless docker has a different default setting than nerdctl, we should just let nerdctl set the default